example/src/showcase/swipeable/AppleStyleSwipeableRow.tsx

```diff
@@ -96,7 +96,13 @@ export default class AppleStyleSwipeableRow extends Component {
         leftThreshold={30}
         rightThreshold={40}
         renderLeftActions={this.renderLeftActions}
-        renderRightActions={this.renderRightActions}>
+        renderRightActions={this.renderRightActions}
+        onSwipeableOpen={(direction) => {
+          console.log(`Opening swipeable to the ${direction}`);
```

Comment:
A very minor nitpick, could you change this message in some way, for example:
```suggestion
          console.log(`Opening swipeable from the ${direction}`);
```
I was confused for a moment when I swiped from left to right and the console log said that it was opening to the left.

Reply:
Sure, will do.

Reply:
Fixed in 5438419.